packages/forklift-console-plugin/src/plans/create/constants.ts

```diff
@@ -0,0 +1,42 @@
+import { t } from 'src/utils/i18n';
+
+export enum PlanWizardStepId {
+  BasicSetUp = 'basic-set-up',
+  General = 'general',
+  VirtualMachines = 'virtual-machines',
+  NetworkMapping = 'network-mapping',
+  StorageMapping = 'storage-mapping',
+  MigrationType = 'migration-type',
+  AdditionalSetUp = 'additional-set-up',
+  OtherSettings = 'other-settings',
+  Hooks = 'hooks',
+  ReviewAndCreate = 'review-and-create',
+}
+
+export const planStepNames: Record<PlanWizardStepId, string> = {
+  [PlanWizardStepId.BasicSetUp]: t('Basic set up'),
+  [PlanWizardStepId.General]: t('General'),
+  [PlanWizardStepId.VirtualMachines]: t('Virtual machines'),
+  [PlanWizardStepId.NetworkMapping]: t('Network mapping'),
+  [PlanWizardStepId.StorageMapping]: t('Storage mapping'),
+  [PlanWizardStepId.MigrationType]: t('Migration type'),
+  [PlanWizardStepId.AdditionalSetUp]: t('Additional set up'),
+  [PlanWizardStepId.OtherSettings]: t('Other settings (optional)'),
+  [PlanWizardStepId.Hooks]: t('Hooks (optional)'),
+  [PlanWizardStepId.ReviewAndCreate]: t('Review and create'),
+};
+
+export const planStepIndexes = Object.values(PlanWizardStepId).reduce(
+  (acc, stepId, index) => ({ ...acc, [stepId]: index + 1 }),
+  {},
+);
+
+export const defaultCurrentStep = {
+  id: PlanWizardStepId.General,
+  parentId: PlanWizardStepId.BasicSetUp,
+  name: PlanWizardStepId.General,
+  index: planStepIndexes[PlanWizardStepId.General],
+};
+
+export const warmMigrationLearnMoreLink =
```

Comment:
lets create a shared location for all docs links, so doc team will have an easy way and a single file to comment on links changes/fixes. something like utils/docLinks.ts